test/lib/test-env.js

```diff
@@ -0,0 +1,8 @@
+'use strict';
+
```

Comment:
good job mocking your env vars... make sure that you never have the APP_SECRET be the same as this file in production